### ISSUE TITLE
WARNING: An illegal reflective access operation has occurred

### ISSUE THREAD

**Issue by alias454** [Graylog2/graylog2-server#7223]
When using OpenJDK 11 on CentOS 7, I see warnings about illegal reflective access operations that show up after startup.

Output from systemctl -l status graylog-server:
```
● graylog-server.service - Graylog server
   Loaded: loaded (/usr/lib/systemd/system/graylog-server.service; enabled; vendor preset: disabled)
   Active: active (running) since Wed 2020-01-15 00:35:06 CST; 3 days ago
     Docs: http://docs.graylog.org/
 Main PID: 17354 (graylog-server)
   CGroup: /system.slice/graylog-server.service
           ├─17354 /bin/sh /usr/share/graylog-server/bin/graylog-server
           └─17356 /usr/bin/java -Xms2g -Xmx2g -XX:NewRatio=1 -server -XX:+ResizeTLAB -XX:+CMSConcurrentMTEnabled -XX:+CMSClassUnloadingEnabled -XX:-OmitStackTraceInFastThrow -jar -Dlog4j.configurationFile=file:///etc/graylog/server/log4j2.xml -Djava.library.path=/usr/share/graylog-server/lib/sigar -Dgraylog2.installation_source=rpm /usr/share/graylog-server/graylog.jar server -f /etc/graylog/server/server.conf -np

Jan 15 00:35:06 graylog systemd[1]: Started Graylog server.
Jan 15 00:35:07 graylog graylog-server[17354]: WARNING: sun.reflect.Reflection.getCallerClass is not supported. This will impact performance.
Jan 15 00:35:12 graylog graylog-server[17354]: WARNING: An illegal reflective access operation has occurred
Jan 15 00:35:12 graylog graylog-server[17354]: WARNING: Illegal reflective access by com.google.inject.assistedinject.FactoryProvider2$MethodHandleWrapper (file:/usr/share/graylog-server/graylog.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
Jan 15 00:35:12 graylog graylog-server[17354]: WARNING: Please consider reporting this to the maintainers of com.google.inject.assistedinject.FactoryProvider2$MethodHandleWrapper
Jan 15 00:35:12 graylog graylog-server[17354]: WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
Jan 15 00:35:12 graylog graylog-server[17354]: WARNING: All illegal access operations will be denied in a future release
```

## Context
I have not done any testing on this and the service seems to startup and run.

## Your Environment
Graylog server 3.1.4+1149fe1
JRE: Oracle Corporation 11.0.5 on Linux 3.10.0-1062.9.1.el7.x86_64
Deployment: rpm
OS: CentOS Linux 7 (Core) (centos)
Arch: amd64

**Comment by radykal-com** [comment]
I think it is related to https://github.com/google/guice/issues/1133 (which doesn't have a fix yet, there is some WIP though)

**Comment by braiam** [comment]
This was fixed in guice via commit [85e30be](https://github.com/google/guice/commit/85e30beafe55551a649025d3c12e831214057412).

**Comment by Seb35** [comment]
This is the same warning as in #9802 (just different Graylog version).
